packages/core/src/browser/quick-open/quick-command-service.ts

```diff
@@ -59,6 +59,12 @@ export class QuickCommandService implements QuickOpenModel, QuickOpenHandler {
     getOptions(): QuickOpenOptions {
         return { fuzzyMatchLabel: true };
     }
+
+    private compareCommands(a: Command, b: Command): number {
```

Comment:
Let's move this as a comparator function under the `Command` namespace? What do you think?

At least, please make it `undefined`-safe. Yes, I know, it wasn't `undefined`-safe earlier either but it was a local code, now, you moved the logic into a method. Please fix it.

Reply:
> Let's move this as a comparator function under the `Command` namespace? What do you think?
> 
> At least, please make it `undefined`-safe. Yes, I know, it wasn't `undefined`-safe earlier either but it was a local code, now, you moved the logic into a method. Please fix it.

Sure @kittaakos, I'll make the necessary changes